frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -129,70 +137,107 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
   };
 
   return (
-    <form onSubmit={submit} className="modal-content modal-content--no-inner-scroll">
-      <ModalTitle>{t('ceph-storage-plugin~Add Capacity')}</ModalTitle>
-      <ModalBody>
-        <Trans t={t} ns="ceph-storage-plugin" values={{ name }}>
-          Adding capacity for <strong>{{ name }}</strong>, may increase your expenses.
-        </Trans>
-        <div className="ceph-add-capacity__modal">
-          <div
-            className={classNames('ceph-add-capacity__sc-dropdown', {
-              'ceph-add-capacity__sc-dropdown--margin': !isNoProvionerSC,
-            })}
+    <Flex direction={{ default: 'column' }}>
+      <Form className="pf-u-background-color-100 pf-u-pt-xl pf-u-pl-xl">
+        <FlexItem>
+          <Title headingLevel="h4" size="xl">
```

Comment:
We can use `ModalTitle`, `ModalBody` and `ModalFooter` they have predefined css and are set as per conventions.